File: authorization-service/src/main/java/com/mazeltov/authorization/service/config/Configure.java
package com.mazeltov.authorization.service.config;

import com.mazeltov.common.spring.*;
import org.springframework.context.annotation.*;

@Configuration
public class Configure {

    @Bean
    public InjectLoggerAnnotationBeanPostProcessor injectLoggerAnnotationBeanPostProcessor(){
        return new InjectLoggerAnnotationBeanPostProcessor();
    }
}

File: product-service/src/main/kotlin/com/mazeltov/product/service/config/Configure.kt
package com.mazeltov.product.service.config

import com.mazeltov.common.spring.*
import org.springframework.beans.factory.config.*
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.context.annotation.*

@Configuration
@EnableConfigurationProperties
@PropertySources(value = [
    PropertySource("classpath:routs.yml")]
)
class Configure {


    @Bean
    fun injectLoggerAnnotationBeanPostProcessor(): BeanPostProcessor {
        return InjectLoggerAnnotationBeanPostProcessor()
    }

}
